### ISSUE TITLE
Got exception: IllegalArgumentException: Missing closing brace: }

### ISSUE THREAD

**Issue by genius1wjc** [square/phrase#29]
Hi there,

The following string and the invocation of Phrase got me this exception every time. Could someone take a look and tell me what went wrong? Thanks! :
`<string name="example_string">{aaa} (Now {bbb})</string>`

```
Phrase.from(this, R.string.example_string)
                    .put("aaa", getAaa())
                    .put("bbb", getBbb())
                    .format()
```

This is the exception:

```
Exploded trying to parse content: java.lang.IllegalArgumentException: Missing closing brace: }
    at com.squareup.phrase.Phrase.key(Phrase.java:245)
    at com.squareup.phrase.Phrase.token(Phrase.java:221)
    at com.squareup.phrase.Phrase.<init>(Phrase.java:204)
    at com.squareup.phrase.Phrase.from(Phrase.java:112)
    at com.squareup.phrase.Phrase.from(Phrase.java:103)
    at com.squareup.phrase.Phrase.from(Phrase.java:94)
```

**Comment by genius1wjc** [comment]
I am using Phrase 1.1.0

**Comment by JoaquimLey** [comment]
For anyone visiting this issue, I just solved it by using `snake_case`, the error message is quite ambiguous imo but if someone has a key on phrase `{writtenLikeThis}` try to change it to `{written_like_this}`

**Comment by pforhan** [comment]
I want to say we still haven't released my error message improvement commits, we should see about getting a new release out.

**Comment by chrislacy** [comment]
I just was struck by this issue when using a string of `"{app_name_1} and {app_name_2}"`.

Removing the numbers from the keys fixed it, but certainly if @pforhan's aa3ec64c8260cc99fc59867ce954c449bb5e2dfc changes were released it would have saved me some time.

**Comment by edenman** [comment]
This should be fixed in 1.2.0
